### ISSUE TITLE
[Question]: 微调UIE完全没有报错，直接退出。已经把batchsize调成1且用的uie-nano

### ISSUE THREAD

**Issue by Fakezzy** [PaddlePaddle/PaddleNLP#6087]
### 请提出你的问题

python finetune.py   --device gpu --logging_steps 1 --seed 42 --model_name_or_path uie-nano --output_dir ./checkpoint/model_best_new --train_path data/train.txt --dev_path data/dev.txt  --max_seq_length 256  --per_device_eval_batch_size 1 --per_device_train_batch_size 1 --num_train_epochs 10 --learning_rate 1e-5 --label_names "start_positions" "end_positions"  --do_train --do_eval --export_model_dir ./checkpoint/model_best_new                                    
G:\miniconda3\lib\site-packages\_distutils_hack\__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
[2023-06-02 18:09:03,991] [ WARNING] - evaluation_strategy reset to IntervalStrategy.STEPS for do_eval is True. you can also set evaluation_strategy='epoch'.
[2023-06-02 18:09:03,992] [    INFO] - using `logging_steps` to initialize `eval_steps` to 1
[2023-06-02 18:09:03,992] [    INFO] - The default value for the training argument `--report_to` will change in v5 (from all installed integrations to none). In v5, you will need to use `--report_to all` to get the same behavior as now. You should start updating your code and make this info disappear :-).
[2023-06-02 18:09:03,992] [    INFO] - ============================================================
[2023-06-02 18:09:03,993] [    INFO] -      Model Configuration Arguments
[2023-06-02 18:09:03,993] [    INFO] - paddle commit id              :0e92adceae06b6b7463f2dc7790ffb0601730009
[2023-06-02 18:09:03,993] [    INFO] - export_model_dir              :./checkpoint/model_best_new
[2023-06-02 18:09:03,993] [    INFO] - model_name_or_path            :uie-nano
[2023-06-02 18:09:03,994] [    INFO] - multilingual                  :False
[2023-06-02 18:09:03,994] [    INFO] -
[2023-06-02 18:09:03,994] [    INFO] - ============================================================
[2023-06-02 18:09:03,994] [    INFO] -       Data Configuration Arguments
[2023-06-02 18:09:03,995] [    INFO] - paddle commit id              :0e92adceae06b6b7463f2dc7790ffb0601730009
[2023-06-02 18:09:03,995] [    INFO] - dev_path                      :data/dev.txt
[2023-06-02 18:09:03,995] [    INFO] - dynamic_max_length            :None
[2023-06-02 18:09:03,995] [    INFO] - max_seq_length                :256
[2023-06-02 18:09:03,995] [    INFO] - train_path                    :data/train.txt
[2023-06-02 18:09:03,996] [    INFO] -
[2023-06-02 18:09:03,996] [ WARNING] - Process rank: -1, device: gpu, world_size: 1, distributed training: False, 16-bits training: False
[2023-06-02 18:09:03,996] [    INFO] - We are using <class 'paddlenlp.transformers.ernie.tokenizer.ErnieTokenizer'> to load 'uie-nano'.
[2023-06-02 18:09:03,997] [    INFO] - Already cached C:\Users\hexinkaifa\.paddlenlp\models\uie-nano\ernie_3.0_nano_zh_vocab.txt
[2023-06-02 18:09:04,012] [    INFO] - tokenizer config file saved in C:\Users\hexinkaifa\.paddlenlp\models\uie-nano\tokenizer_config.json
[2023-06-02 18:09:04,013] [    INFO] - Special tokens file saved in C:\Users\hexinkaifa\.paddlenlp\models\uie-nano\special_tokens_map.json
[2023-06-02 18:09:04,015] [    INFO] - Model config ErnieConfig {
  "attention_probs_dropout_prob": 0.1,
  "enable_recompute": false,
  "fuse": false,
  "hidden_act": "gelu",
  "hidden_dropout_prob": 0.1,
  "hidden_size": 312,
  "initializer_range": 0.02,
  "intermediate_size": 1248,
  "layer_norm_eps": 1e-12,
  "max_position_embeddings": 2048,
  "model_type": "ernie",
  "num_attention_heads": 12,
  "num_hidden_layers": 4,
  "pad_token_id": 0,
  "paddlenlp_version": null,
  "pool_act": "tanh",
  "task_id": 0,
  "task_type_vocab_size": 16,
  "type_vocab_size": 4,
  "use_task_id": true,
  "vocab_size": 40000
}

W0602 18:09:04.173038  7828 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 11.7, Runtime API Version: 10.2
W0602 18:09:04.181079  7828 gpu_resources.cc:91] device: 0, cuDNN Version: 7.6.

显卡是3070，解析后的train.txt数据大概20000条。

**Comment by linjieccc** [comment]
你好，请尝试下安装对应cuda版本的paddle

```shell
python -m pip install paddlepaddle-gpu==2.4.2.post117 -f https://www.paddlepaddle.org.cn/whl/linux/mkl/avx/stable.html
```

**Comment by Fakezzy** [comment]
> 你好，请尝试下安装对应cuda版本的paddle
> 
> ```shell
> python -m pip install paddlepaddle-gpu==2.4.2.post117 -f https://www.paddlepaddle.org.cn/whl/linux/mkl/avx/stable.html
> ```

我将linux替换成windows 然后通过你这个命令安装了对应的包就解决了问题  感谢
